tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -646,6 +754,81 @@ def do_testharness(self, protocol, url, timeout):
 
         return rv
 
+    def bidi_deserialize(self, bidi_value):
```

Comment:
It would be very nice to use the existing WebDriver library rather than duplicating the functionality here.

Reply:
Do we have the script deserialization implemented in WPT?